yt_dlp/extractor/lsm.py

```diff
@@ -267,12 +269,27 @@ def _real_extract(self, url):
 
         data = self._search_nuxt_data(
             self._fix_nuxt_data(webpage), video_id, context_name='__REPLAY__')
+        playback_type = traverse_obj(data, ('playback', 'type'))
+
+        if playback_type == 'playable_audio_lr':
+            playback_data = {
+                'formats': self._extract_m3u8_formats(
+                    traverse_obj(data, ('playback', 'service', 'hls_url', {url_or_none})), video_id),
```

Comment:
same here
```suggestion
                'formats': self._extract_m3u8_formats(data['playback']['service']['hls_url'], video_id),
```

and do we want to try extracting subs from the m3u8 too?

Reply:
I don't think it's necessary. These are always audio only and I have not seen an example with with any sort of transcript.